contracts/ClaimsManager.sol

```diff
@@ -35,6 +35,7 @@ contract ClaimsManager is
     address public override api3Pool;
     uint256 public override mediatorResponsePeriod;
     uint256 public override claimantResponsePeriod;
+    uint256 public override claimValidityPeriod = 259200;
```

Comment:
Can you make this a parameter to be specified during construction like the above?